src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -9,6 +9,7 @@ import { MessageToDisplay } from "types/chat";
 import { useProfileModalControls } from "hooks/useProfileModalControls";
 
 import { UserAvatar } from "components/atoms/UserAvatar";
+import { getLinkFromText } from "utils/getLinkFromText";
```

Comment:
Re-arranging import order to match our preferred style (types, utils, hooks, components, local `./` imports, etc)

```suggestion
import { getLinkFromText } from "utils/getLinkFromText";

import { useProfileModalControls } from "hooks/useProfileModalControls";

import { UserAvatar } from "components/atoms/UserAvatar";
```